documentation/docs/contributing/local-deployment.md

```diff
@@ -8,7 +8,10 @@ sidebar_position: 3
 ---
 
 If you are integrating a new scanner or hook and want to test from a local build, this document will guide you through it.
-This guide assumes that you have your Kubernetes cluster already up and running and that we can work in your default namespace. If not, check out the [installation](/docs/getting-started/installation/) for more information.
+:::note
+Currently, we only offer the option of local deployment using a [kind](https://kind.sigs.k8s.io/) cluster.
+:::
+This guide assumes that you have your Kubernetes cluster, installed via [kind](https://kind.sigs.k8s.io/), already up and running, and that we can work in your default namespace. If not, check out the [installation](/docs/getting-started/installation/) for more information.
```

Comment:
The installation page doesn't describe how to setup the kind cluster. I'd tweak it like this

```suggestion
For simplicity sake this guide is only written for local cluster stups using [kind](https://kind.sigs.k8s.io/).
Other setups (e.g. minikube, docker-desktop integrated kubernetes) are possible but might require extra setup & tweaking effort to get running with our make files.

## Kind Cluster Setup

1. Create the kind cluster `kind create cluster`
2. Install the secureCodeBox operator. See [installation](/docs/getting-started/installation/)
3. Create the `integration-tests` namespace. `kubectl create namespace integration-tests`
```